db/migrate/20180304132024_competitors_migrate_gender_column.rb

```diff
@@ -0,0 +1,14 @@
+class CompetitorsMigrateGenderColumn < ActiveRecord::Migration[5.1]
+  def change
+    migrate_competitors(male: true, gender: 'male')
+    migrate_competitors(male: false, gender: 'female')
+  end
+
+  private
+
+  def migrate_competitors(male:, gender:)
+    Competitor.where(male: male).find_in_batches do |competitors|
+      Competitor.where(id: competitors.map(&:id)).update_all(gender: gender)
```

Comment:
This feels a bit weird to me.  Can't you do this in one SQL command instead?

```ruby
Competitor.where(male: male).update_all(gender: gender)
```

Is the idea to give your sql server a break during this migration?

Reply:
Yea, total premature optimisation for this tiny database, lol